components/engine/libcontainerd/client_linux.go

```diff
@@ -330,6 +332,14 @@ func (clnt *client) restore(cont *containerd.Container, lastEvent *containerd.Ev
 		return fmt.Errorf("container %s is already active", containerID)
 	}
 
+	if errUpgrade := v17_06_1.Upgrade(
```

Comment:
by this time, it's too late if I'm not mistaken.

`containerd` on restart tries to load all of those files and would have failed to unmarshal them